### ISSUE TITLE
PHP 8.2 compatibility

### ISSUE THREAD

**Issue by captainark** [operasoftware/dns-ui#201]
Hello,

I've just updated my PHP installation to 8.2, and I'm now getting a blank page for my dns-ui installation.

I'm running the latest commit from the master branch.

I'm getting the following error in my nginx logs:

```
2023/03/16 16:34:44 [error] 4061153#4061153: *2616128 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught ErrorException: Creation of dynamic property ZoneDirectory::$cache_uid is deprecated in /var/www/dns-ui/model/zonedirectory.php:35
Stack trace:
#0 /var/www/dns-ui/model/zonedirectory.php(35): exception_error_handler()
#1 /var/www/dns-ui/core.php(84): ZoneDirectory->__construct()
#2 /var/www/dns-ui/core.php(56): setup_database()
#3 /var/www/dns-ui/requesthandler.php(19): require('...')
#4 /var/www/dns-ui/public_html/init.php(18): require('...')
#5 {main}
  thrown in /var/www/dns-ui/model/zonedirectory.php on line 35" while reading response header from upstream, client: 169.254.255.10, server: example.com, request: "GET /dns-ui/ HTTP/2.0", upstream: "fastcgi://unix:/run/php/php8.2-fpm.sock:", host: "dns-ui.example.com"
```

**Comment by zerwes** [comment]
for those whoy need a hotfix-release: https://github.com/Rosa-Luxemburgstiftung-Berlin/dns-ui/releases/tag/v0.2.7-203

**Comment by thomas-pike** [comment]
Fixed in 15f0dc29.
